### PR TITLE
[Forwardport] `8.6.2` release notes to `8.7` (#14896)

### PR DESCRIPTION
**Pull request by mashhurs**
Forwardporting `8.6.2` release notes to `8.7`.

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -31,18 +33,48 @@ This section summarizes the changes in the following releases:
...
 === Logstash 8.6.0 Release Notes
 
 [[features-8.6.0]]
 ==== New features and enhancements
 
 * Extends the flow rates introduced to the Node Stats API in 8.5.0 (which included windows for `current` and `lifetime`)
-to include a Technology Preview of several additional windows such as `last_15_minutes`, `last_24_hours`, etc..
-https://github.com/elastic/logstash/pull/14571[#14571]
+  to include a Technology Preview of several additional windows such as `last_15_minutes`, `last_24_hours`, etc..
+  https://github.com/elastic/logstash/pull/14571[#14571]
```
Comment (andsel) [inline docs/static/releasenotes.asciidoc:74]: The spaces and the following weren't present in the origin al PR #14896

**Review by andsel — CHANGES_REQUESTED** [review]
Are the added spaces part of this bakport?

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (mashhurs) [reply docs/static/releasenotes.asciidoc:74]: Yes and I was surprised when cherry picking that somehow this change lost, here is [the origin change](https://github.com/elastic/logstash/pull/14762/files)

Comment (andsel) [reply docs/static/releasenotes.asciidoc:74]: When `8.6.0` release notes was forward ported, those spaces wasn't moved also check [this](https://github.com/elastic/logstash/pull/14833/files#diff-51d73a6b1811f83bad13ef62e2d88207e0f028ef70e3e9b968ea411bc817f69cR42) for example.

Comment (andsel) [reply docs/static/releasenotes.asciidoc:74]: Good catch

**Pull request merged**